src/20230924_AaveV3_Opt_OPRiskParametersUpdate/AaveV3_Optimism_OPRiskParametersUpdate_20230924.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3PayloadOptimism, IEngine, Rates, EngineFlags} from 'aave-helpers/v3-config-engine/AaveV3PayloadOptimism.sol';
+import {AaveV3Optimism, AaveV3OptimismAssets} from 'aave-address-book/AaveV3Optimism.sol';
+
+/**
+ * @title OP Risk Parameters Update
+ * @author Marc Zeller - Aave-Chan Initiative
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x617adb838ce95e319f06f72e177ad62cd743c2fe3fd50d6340dfc8606fbdd0b3
+ * - Discussion: https://governance.aave.com/t/arfc-op-risk-parameters-update-aave-v3-optimism-pool/14633
+ */
+contract AaveV3_Optimism_OPRiskParametersUpdate_20230924 is AaveV3PayloadOptimism {
+  uint256 public constant BORROW_CAP = 500_000;
+  address public constant INTEREST_RATE_STRATEGY = 0x3B57B081dA6Af5e2759A57bD3211932Cb6176997;
+  uint256 public constant DEBT_CEILING = 0;
+
+  function _preExecute() internal override {
```

Comment:
Please change IR strategy update to use ConfigEngine, similar to [this example](https://github.com/bgd-labs/aave-proposals/blob/622f2741d687033ed890d51506bd8c0222fcf883/src/AaveV2PolygonRatesUpdates_20230614/AaveV2PolygonRatesUpdates_20230614.sol#L46)

Reply:
i've rm rateStrategy update, it's not needed